polygeist/lib/Dialect/Polygeist/Transforms/SYCLRaiseHost.cpp

```diff
@@ -1343,6 +1343,130 @@ class RaiseSetNDRange
     Value value;
   };
 };
+
+class RaiseSetCaptured : public OpHostRaisePattern<LLVM::VarAnnotation> {
```

Comment:
I would like a comment that explain what is the role of this class. Arguably we should have such brief descriptions for all the patterns.

Reply:
Added, let me know it's sufficient, please.